docs/internal-documentation/maintenance/Instructions-for-translators.md

```diff
@@ -0,0 +1,405 @@
+  robust.
+* There is also "[GNOME Translation Editor]", a graphical PO editor
+  available for at least Windows and Linux.
+* There is [POEDIT] in free version that Zonemaster translator has reported to
+  work well.
+* There are more tools available, either cloud services or programs
+  for download, and they could be found by searching for "po editor".
+
+
+## Clone preparation
+
+You need a local clone of the repository to work in (see
+"[Cloning a repository]"). You need that for all three repositories
+(Zonemaster-Engine, Zonemaster-CLI and Zonemaster-Backend). Here we use
+Zonemaster-Engine as an example. You do the exact same steps for the other
+two repositories.
+
+* Clone the Zonemaster-Engine repository, unless you already have a clone that
+  you could reuse:
+  ```
+  git clone https://github.com/zonemaster/zonemaster-engine.git
+  ```
+
+* Enter the directory of the clone created above or already existing clone:
+  ```
+  cd zonemaster-engine
+  ```
+
+* If you already have an old clone of Zonemaster-Engine, run an update.
+  ```
+  git fetch --all
+  ```
+
+* Now it is time to connect your own fork of *Zonemaster-Engine* at Github to
+  the created clone, unless you have alreday done that, in case you can skip the
+  next step.
+
+* You have a user name at Github. Here we use "xxxx" as your user name and also
+  the name of the remote in clone on the local machine.
+  ```
+  git remote add xxxx git@github.com:xxxx/zonemaster-engine.git
+  git fetch --all
+  ```
+
+Now you should have updated local clones of all three repositories.
+
+
+## Translation steps
+
+The steps in this section will cover most translation work. The steps are the
+same for all three repositories.
+
+* Check-out the *develop* branch and create a new branch to work in. You can call
+  the new branch whatever you want, but here we use the name
+  "translation-update". If that name is already taken, you have to give it a new
+  name or remove the old branch.
+  ```
+  git checkout origin/develop
+  git checkout -b translation-update
+  ```
+
+* Go to the *share* directory and run the update command for the PO file for
+  the language you are going to work with. Replace "xx" with the language code in
+  question. This should be done every time.
+  ```
+  cd share
+  ./update-po xx.po
+  ```
+
+* The PO file is updated with new *msgids*, if any, and now you can start
+  working with it. Unless this is the first translation you only have to work
+  with updated or untranslated items.
+
+* Update the PO file with the tool of your choice. See above. You can copy
+  the PO file to another computer, edit it there, and then copy it back to
+  your Zonemaster-Engine clone -- or Zonemaster-CLI clone or Zonemaster-Backend
+  clone depending on what repository you do the translation for.
+
+* When doing the update, do not change the *msgid*, only the *msgstr*. The
+  *msgid* cannot be be update in this process. They are the links between the
```

Comment:
Removed extra "be". -- It is still correct that msgid cannot be updated in the translation process. If you update it in the PO file, it will be replaced next time `update-po` is run. Even if it goes all the way to the MO file it will not be an update of the *msgid*.

Reply:
Okay, now I see how you think.